pabi_budget_drilldown_report/wizard/budget_drilldown_report_wizard.py

```diff
@@ -36,6 +36,12 @@ class BudgetDrilldownReportWizard(SearchCommon, models.TransientModel):
 
     @api.onchange('report_type')
     def _onchange_report_type(self):
+        super(BudgetDrilldownReportWizard, self)._onchange_report_type()
+        # Clear Data
+        for field in ['section_id', 'project_id', 'activity_group_id',
+                      'charge_type', 'activity_id']:
+            self['group_by_%s' % (field)] = False
```

Comment:
You can use,
'group_by_%s' % field

Note: To use tuple. If there is 1 value, it should be (field, ). (you will need to add comma). Otherwise, if it is only 1 value, you don't need to use tuple.